src/pr-info.ts

```diff
@@ -520,12 +521,28 @@ function downloadsToPopularityLevel(monthlyDownloads: number): PopularityLevel {
 export async function getOwnersOfPackage(packageName: string, oid: string, fetchFile: typeof defaultFetchFile): Promise<string[] | null | Error> {
     const packageJson = `${oid}:types/${packageName}/package.json`;
     const packageJsonContent = await fetchFile(packageJson, 10240); // grab at most 10k
-    if (packageJsonContent === undefined) return null;
     let packageJsonObj;
-    try {
-        packageJsonObj = JSON.parse(packageJsonContent);
-    } catch (e) {
-        if (e instanceof Error) return new Error(`error parsing owners from package.json: ${e.message}`);
+    if (packageJsonContent !== undefined) {
```

Comment:
Alternatively, I can try the header _first_ and only break out if it fails to parse.